### DISCUSSION TITLE
How to make clear color 1:1 ratio?

### DISCUSSION THREAD

**Discussion by ThreeTwoBravo** [cables-gl/cables_docs#238] [Answers & Questions] [answered]
Hi,

I have a render2texture set at 2048x2048 and a 1:1 RectangleFrame. Whenever I go to fullscreen the clear color goes fullscreen. Anything after the render2texture should be the same ratio, yes?

![image](https://user-images.githubusercontent.com/50957740/167987504-d0dfee22-63d8-49ca-9681-fe6d0a7c8ba3.png)

**Comment by tastez** [comment]
is your fullscreen rectangle set to stretch or fit?

**Reply by ThreeTwoBravo** [reply]
it's set to fit

**Comment by pandrr** [comment, marked as answer]
![Screenshot 2022-05-18 at 08 04 34](https://user-images.githubusercontent.com/98792/168968460-cc677d8c-8113-4b8e-8719-bab39903463e.png)

works fine for me, please publish an example patch or provide more info on what is wrong
